README.md

```diff
@@ -136,20 +141,20 @@ We will set up a deploy key in the Git server that will allow an SSH key we make
     puppet agent -t
     ~~~
 
-5. Navigate back to the **Nodes > Classification** page
- - Near the top of the page select "add a group"
- - Type `role::all_in_one_pe` for the group name
-    - Click the **Add Group** button
- - Click the **add membership rules, classes and variables** link that appears
-    - Below **Pin specific nodes to the group** type your master's FQDN into the box
-       - Click **pin node**
- - Select the **Classes** tab
-    - On the right hand side, click the **Refresh** link
-       - Wait for this to complete
-    - In the **add new classes** box type `role::all_in_one_pe`
-       - Click **add class**
- - **Commit** your changes
-8. On your Puppet master
+5. In the PE console, navigate to **Nodes > Classification**.
+ - Near the top of the page select "add a group".
+ - For the group name enter `role::all_in_one_pe`
+    - Click the **Add Group** button.
+ - Click the **add membership rules, classes and variables** link.
+    - Below **Pin specific nodes to the group** enter your master's FQDN.
```

Comment:
More or less, type = enter, box = field. Either way I don't think anyone's going to misunderstand.